helix-view/Cargo.toml

```diff
@@ -16,6 +16,7 @@ term = ["crossterm"]
 [dependencies]
 bitflags = "2.3"
 anyhow = "1"
+rustix = { version = "0.38", features = ["fs"] }
```

Comment:
I think this should only be a `cfg(unix)' dependency (its possible to make dependencies target specific)